bridges/otelzap/test_helper.go

```diff
@@ -0,0 +1,149 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otelzap // import "go.opentelemetry.io/contrib/bridges/otelzap"
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"go.uber.org/zap/zapcore"
+
+	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/log/embedded"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+)
+
+// embeddedLogger is a type alias so the embedded.Logger type doesn't conflict
+// with the Logger method of the recorder when it is embedded.
+type embeddedLogger = embedded.Logger // nolint:unused  // Used below.
+
+// recorder records all [log.Record]s it is ased to emit.
+type recorder struct {
```

Comment:
Can you use https://pkg.go.dev/go.opentelemetry.io/otel/log@v0.1.0-alpha.0.20240409140646-648b40eae158/logtest instead?

You can get it via `go get go.opentelemetry.io/otel/log@v0.1.0-alpha.0.20240409140646-648b40eae158`.